sysutils/apcupsd/src/opnsense/service/templates/OPNsense/Apcupsd/apcupsd.conf

```diff
@@ -0,0 +1,357 @@
+{% if helpers.exists('OPNsense.apcupsd.general.Enabled') and OPNsense.apcupsd.general.Enabled == "1" %}
```

Comment:
don't think clearing the configuration file is necessary when we don't start the service. may have unintended side effects since start/stop/restart are decoupled from configuration creation